Sources/JWT/Signers/RSASigner+X509.swift

```diff
@@ -0,0 +1,30 @@
+import CTLS
+
+typealias CX509Key = UnsafeMutablePointer<X509>
+
+extension RSAKey {
+    public init(x509Cert cert: Bytes) throws {
+        guard
+            let cert = cert.withUnsafeBufferPointer({ rawKeyPointer -> CX509Key? in
```

Comment:
I haven't tested the PR, but judging by eye, the X509 structure returned by `d2i_X509` is leaked.